src/components/styled/PaperInput.ts

```diff
@@ -14,8 +14,8 @@ const PaperInput = styled(Paper)({
   borderStyle: "solid",
   borderColor: theme.colors.grey400,
   borderRadius: "9px",
-  boxShadow: "inset 0px 3px 6px #000000D0",
```

Comment:
I removed the #rrggbbaa hex color notation as it was doing very little and browsers are barely adopting it from what i [see here](https://caniuse.com/css-rrggbbaa)

lmk if i should add it back though